packages/forms-web-app/src/controllers/selected-appeal/representations/index.js

```diff
@@ -0,0 +1,72 @@
+const { formatHeadlineData } = require('@pins/common');
+const { VIEW } = require('../../../lib/views');
+const { formatTitleSuffix } = require('../../../lib/selected-appeal-page-setup');
+const { getDepartmentFromCode } = require('../../../services/department.service');
+const { REPRESENTATION_TYPES } = require('@pins/common/src/constants');
+const {
+	filterRepresentationsBySubmittingParty,
+	formatRepresentationHeading,
+	formatRepresentations
+} = require('../../../lib/representation-functions');
+
+/**
+ * @typedef {import('@pins/common/src/constants').AppealToUserRoles} AppealToUserRoles
+ * @typedef {import('@pins/common/src/constants').LpaUserRole} LpaUserRole
+ * @typedef {import('@pins/common/src/constants').RepresentationTypes} RepresentationTypes
+ */
+
+/**
+ * @param {string} url
+ * @returns {AppealToUserRoles|LpaUserRole|null}
+ */
+
+/**
+ * @typedef {Object} RepresentationParams
+ * @property {AppealToUserRoles|LpaUserRole} userType // the user
+ * @property {RepresentationTypes} representationType //
+ * @property {AppealToUserRoles|LpaUserRole} submittingParty  // the party submitting the representation
+ */
+
+/**
+ * Shared controller for /appeals/:caseRef/lpa-statement, /manage-appeals/:caseRef/statement
+ * @param {RepresentationParams} representationParams
+ * @param {string} layoutTemplate - njk template to extend
+ * @returns {import('express').RequestHandler}
+ */
+exports.get = (representationParams, layoutTemplate = 'layouts/no-banner-link/main.njk') => {
```

Comment:
can we get some tests for this controller please

Reply:
Added a test, will look to expand it as more routes get onboarded